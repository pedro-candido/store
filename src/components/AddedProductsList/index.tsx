import { FlashList } from '@shopify/flash-list';

import AddedProduct from '@components/AddedProduct';
import { useStore } from '@store/globalStore';

function AddedProductsList() {
  const { productsChosen } = useStore();

  return (
    <>
      <FlashList
        data={productsChosen}
        estimatedItemSize={20}
        renderItem={({ item }) => <AddedProduct {...item} />}
      />
    </>
  );
}

export default AddedProductsList;
