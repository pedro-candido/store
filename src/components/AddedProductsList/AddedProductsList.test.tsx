import React from 'react';

import { render } from '@testing-library/react-native';

import AddedProductsList from './index';

describe('AddedProductsList', () => {
  it('should match snapshot', () => {
    const testRenderer = render(<AddedProductsList />);

    expect(testRenderer.toJSON).toMatchSnapshot();
  });
});
