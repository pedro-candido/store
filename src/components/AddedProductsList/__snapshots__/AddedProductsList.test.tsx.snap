// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddedProductsList should match snapshot 1`] = `[Function]`;
